transformer_hw_servos:
  ros__parameters:
    gpiochip_name: "gpiochip4"
    gpio_servo0: 6
    gpio_servo1: 18
    period_us: 20000
    initial_us: 1500
    initial_us_servo0: 1550
    initial_us_servo1: 1000
    min_us: 600
    max_us: 3000
    min_us_servo0: 1550
    min_us_servo1: 1100
    max_us_servo0: 3000
    max_us_servo1: 3000
    use_realtime: true
    rt_priority: 90
    cpu_affinity: 2
    tail_spin_us: 100
    lock_memory: true
    prefault_mb: 64
  # New: per-servo start offset in microseconds and number of cycles (0=infinite)
  servo_offset_servo0_us: 0
  servo_offset_servo1_us: 0
  servo_cycles: 0